docs/source/backends/openwrt.rst

```diff
@@ -598,43 +598,68 @@ Interfaces of type ``bridge`` contains options that are specific for network bri
+|                             |         |           | +===========+=========+====================================================================+ |
+|                             |         |           | | ``vlan``  | integer | VLAN ID                                                            | |
+|                             |         |           | +-----------+---------+--------------------------------------------------------------------+ |
+|                             |         |           | | ``ports`` | list    | A list of ``dict`` defining interfaces participating in the VLAN   | |
+|                             |         |           | |           |         |                                                                    | |
+|                             |         |           | |           |         | +-----------------+---------+------------------------------------+ | |
+|                             |         |           | |           |         | | key name        | type    | allowed values                     | | |
+|                             |         |           | |           |         | +=================+=========+====================================+ | |
+|                             |         |           | |           |         | | ``ifname``      | string  | interface name (this interface     | | |
+|                             |         |           | |           |         | |                 |         | should be a bridge member)         | | |
+|                             |         |           | |           |         | +-----------------+---------+------------------------------------+ | |
+|                             |         |           | |           |         | | ``tagging``     | string  | whether the port is tagged (``t``) | | |
+|                             |         |           | |           |         | |                 |         | or untagged (``u``)                | | |
+|                             |         |           | |           |         | +-----------------+---------+------------------------------------+ | |
+|                             |         |           | |           |         | | ``primary_vid`` | boolean | whether the current VLAN should be | | |
+|                             |         |           | |           |         | |                 |         | used for all untagged incoming     | | |
+|                             |         |           | |           |         | |                 |         | traffic on this interface          | | |
+|                             |         |           | |           |         | +-----------------+---------+------------------------------------+ | |
+|                             |         |           | +-----------+---------+--------------------------------------------------------------------+ |
++-----------------------------+---------+-----------+----------------------------------------------------------------------------------------------+
```

Comment:
Why didn't you add any example here?
We should provide an example for at least the configurations covered by tests.